Webview tests are skipped

Couple of milestones ago I have skipped these tests becuase they were failing. I am not sure if I created the issue at the moment but I am doing that now. Sorry if I did not ping sooner.

Code pointer
https://github.com/microsoft/vscode/blob/ca370bdb0aef33613cbaced701c38281cf851e19/extensions/vscode-api-tests/src/singlefolder-tests/webview.test.ts#L20

I had to revert this change: https://github.com/microsoft/vscode/commit/6bd7b70515d546aba9010d2a0249a1a19205822f

I think they cause our browser tests to hang/fail:

```
 vscode API - webview
    1) webviews should be able to send and receive messages
    √ webviews should not have scripts enabled by default (1001ms)
    2) webviews should update html
    - webviews should preserve vscode API state when they are hidden
    3) webviews should preserve their context when they are moved between view columns
    4) webviews with retainContextWhenHidden should preserve their context when they are hidden
Terminate batch job (Y/N)? 
##[error]The task has timed out.
```

https://dev.azure.com/monacotools/Monaco/_build/results?buildId=99855&view=logs&j=672276a2-8d3a-5fab-615d-090c51352f92&t=354d3e13-3534-5793-0b68-b00f8c1d36f3&l=136

--- a/extensions/vscode-api-tests/src/singlefolder-tests/webview.test.ts
+++ b/extensions/vscode-api-tests/src/singlefolder-tests/webview.test.ts
@@ -17,7 +17,7 @@ function workspaceFile(...segments: string[]) {
 
 const testDocument = workspaceFile('bower.json');
 
-suite.skip('vscode API - webview', () => {
+suite('vscode API - webview', () => {
 	const disposables: vscode.Disposable[] = [];
 
 	function _register<T extends vscode.Disposable>(disposable: T) {
@@ -400,7 +400,7 @@ suite.skip('vscode API - webview', () => {
 		});
 	}
 
-	test('webviews should transfer ArrayBuffers to and from webviews', async () => {
+	test.skip('webviews should transfer ArrayBuffers to and from webviews', async () => {
 		const webview = _register(vscode.window.createWebviewPanel(webviewId, 'title', { viewColumn: vscode.ViewColumn.One }, { enableScripts: true, retainContextWhenHidden: true }));
 		const ready = getMessage(webview);
 		webview.webview.html = createHtmlDocumentWithBody(/*html*/`
@@ -451,7 +451,7 @@ suite.skip('vscode API - webview', () => {
 		}
 	});
 
-	test('webviews should transfer Typed arrays to and from webviews', async () => {
+	test.skip('webviews should transfer Typed arrays to and from webviews', async () => {
 		const webview = _register(vscode.window.createWebviewPanel(webviewId, 'title', { viewColumn: vscode.ViewColumn.One }, { enableScripts: true, retainContextWhenHidden: true }));
 		const ready = getMessage(webview);
 		webview.webview.html = createHtmlDocumentWithBody(/*html*/`

--- a/src/vs/workbench/services/environment/browser/environmentService.ts
+++ b/src/vs/workbench/services/environment/browser/environmentService.ts
@@ -233,7 +233,7 @@ export class BrowserWorkbenchEnvironmentService implements IWorkbenchEnvironment
 			|| 'https://{{uuid}}.vscode-webview.net/{{quality}}/{{commit}}/out/vs/workbench/contrib/webview/browser/pre/';
 
 		return endpoint
-			.replace('{{commit}}', this.productService.commit || '97740a7d253650f9f186c211de5247e2577ce9f7')
+			.replace('{{commit}}', this.payload?.get('webviewExternalEndpointCommit') ?? this.productService.commit ?? '97740a7d253650f9f186c211de5247e2577ce9f7')
 			.replace('{{quality}}', this.productService.quality || 'insider');
 	}
 

--- a/test/integration/browser/src/index.ts
+++ b/test/integration/browser/src/index.ts
@@ -46,7 +46,7 @@ async function runTestsInBrowser(browserType: BrowserType, endpoint: url.UrlWith
 	const testFilesUri = url.format({ pathname: URI.file(path.resolve(optimist.argv.extensionTestsPath)).path, protocol, host, slashes: true });
 
 	const folderParam = testWorkspaceUri;
-	const payloadParam = `[["extensionDevelopmentPath","${testExtensionUri}"],["extensionTestsPath","${testFilesUri}"],["enableProposedApi",""]]`;
+	const payloadParam = `[["extensionDevelopmentPath","${testExtensionUri}"],["extensionTestsPath","${testFilesUri}"],["enableProposedApi",""],["webviewExternalEndpointCommit","5319757634f77a050b49c10162939bfe60970c29"]]`;
 
 	await page.goto(`${endpoint.href}&folder=${folderParam}&payload=${payloadParam}`);
 

